index.html

```diff
@@ -28,7 +28,7 @@ <h2 class="sd">
                 don’t hestiate to contact me.
             </p>
             <ul class="icon-list">
-                <li><a href="https://twitter.com/richardfriday14"><img src="./images/twitter.png" alt="twitter"></a></li>
+                <li><a href="https://twitter.com/richardfriday14" target="blank"><img src="./images/twitter.png" alt="twitter"></a></li>
```

Comment:
- [ ] On your form, all your input fields elements must be wrapped on the form element.  The submit button too.
- [ ] Please, ensure the form works properly and sends the data correctly.

## Explanation

This is the reason why the form does not work properly.  You need to wrap all the elements of the form, inside the `<form>` tag.

## Hints

I managed to make it work like this:

```HTML

      <form
        class="form-style"
        action="https://formspree.io/f/mnqwzknj"
        method="POST"
      >
        <input type="text" maxlength="30" placeholder="Full Name" required />

        <input type="email" name="email" placeholder="Enter address" required />

        <textarea
          class="text"
          name="msg"
          maxlength="500"
          placeholder="Enter text here"
          required
        ></textarea>

        <div class="button-cont">
          <button type="submit" class="potfolio-but">Get in Touch</button>
        </div>
      </form>

```

Reply:
done